README.md

```diff
@@ -65,6 +65,15 @@ The quick start describes how to install and use the operator `mesh` CLI command
 
 ### Installation
```

Comment:
Why is this called installation, if its about building the repo?

Reply:
good point, addressed.